src/motion/features/layout/Animate.tsx

```diff
@@ -61,6 +62,8 @@ class Animate extends React.Component<AnimateProps> {
         visualElement.enableLayoutProjection()
         this.unsubLayoutReady = visualElement.onLayoutUpdate(this.animate)
         visualElement.layoutSafeToRemove = () => this.safeToRemove()
+
+        addScaleCorrection(defaultScaleCorrectors)
```

Comment:
What's this change doing? Did we remove scale correction from somewhere else and are now adding it back to satisfy the bundlers? Hard to see in the big pr with lots of files moving around

Reply:
Basically this is dynamically loading scale correction with layout animations rather than having it synchronously loaded into the main bundle where it might never be used.